robocop/checkers/naming.py

```diff
@@ -676,55 +677,51 @@ def find_similar_variables(self, tokens, node):
 class DeprecatedStatementChecker(VisitorChecker):
     """Checker for deprecated statements."""
 
-    reports = (
-        "deprecated-statement",
-    )
-    # deprecated:alternative
-    deprecated_statements_rf3 = {}
-    deprecated_statements_rf4 = {"runkeywordunless": "IF"}
```

Comment:
So this rule should no longer be supported for RF 4?

Reply:
Good question actually - for Run Keyword If there is ``if-can-be-used`` rule (that works only for ==4.*). There is nothing for ``Run Keyword Unless``. So I can at least make this rule work for 4.0